docs/pages/api.rst

```diff
@@ -453,8 +453,7 @@ contains
 Check if the collection contains one or more values.
 
 .. warning:: The `values` parameter is variadic and will be evaluated as a logical ``OR``.
-             If you're looking for a logical ``AND``, you have to make separate calls to this method;
-             note the calls cannot be in succession because the collection will contain a boolean after the first call.
+             If you're looking for a logical ``AND``, you have to make separate calls to this method.
```

Comment:
I think we can get rid of this line, it doesn't make sense actually.

Reply:
you're right, I simplified this for the other operations that return booleans. It's quite obvious that separate calls should be made if you're trying to test multiple things as an `AND` if the operation returns a boolean 😄